### ISSUE TITLE
[Package Request] - Title

### ISSUE THREAD

**Issue by ahoward-ch** [amazonlinux/amazon-linux-2023#534]
**What package is missing from Amazon Linux 2023? Please describe and include package name.**
package: tesseract
description: ocr package containing many language models

**Is this an update to existing package or new package request?**
New package request

**Is this package available in Amazon Linux 2? If it is available via external sources such as EPEL, please specify.**
tesseract is available in Amazon Linux 2 via amazon-linux-extras

**Any additional information you'd like to include. (use-cases, etc)**
Vital for creating containers to handle small scale ocr in batches.

**Comment by stewartsmith** [comment]
Specifically, this is available via EPEL (https://src.fedoraproject.org/rpms/tesseract shows 3.04 in EPEL)

**Comment by ahoward-ch** [comment]
I didn't seem able to get EPEL in amazon-linux-2023, but that might simply be because I dont know how. 

Or are you simply adding context that tesseract 3 is available in EPEL as an extension of the 3rd answer in the original post? 

For what its worth, tesseract 3 is wildly out of date - tesseract 4 and above switched to using far superior language models and really should be the minimum requirement for tesseract now. Tesseract 4 is what is available in Amazon Linux 2 extras so a like for like would really be needed.

**Comment by schembor** [comment]
> For what its worth, tesseract 3 is wildly out of date 

Yes - Tesseract 3 isn't even compatible with most packages which are used to interact with tesseract, like pytesseract. Also was unable to get EPEL in AL 2023.

**Comment by stewartsmith** [comment]
The note about it being in EPEL was to differentiate this from a package in AL2 that isn't in AL2023.

**Comment by jrozner** [comment]
With EPEL no longer being supported, is it possible to get this added? Right now there is no easy way to do this.

**Comment by ericchen99** [comment]
Threw me off for a while since Tesseract was working fine for us with AL2.

Couldn't find another way to do this other than to build a custom base image where I built Leptonica and Tesseract from source.

**Comment by SreeSunku** [comment]
Thank you for your patience. This package is now available as part of the SPAL Repository for AL2023. Please use these resources to familiarize yourself on installing SPAL repository and the packages.  [SPAL Technical Documentation](https://docs.aws.amazon.com/linux/al2023/ug/configure-spal-repository.html), [SPAL list of packages](https://docs.aws.amazon.com/linux/al2023/release-notes/al-2023-spal-packages.html).
